Source/AliveLibAO/SecurityClaw.cpp

```diff
@@ -107,7 +107,7 @@ SecurityClaw* SecurityClaw::ctor_418A70(Path_SecurityClaw* pTlv, int tlvInfo)
     {
         pClaw->ctor();
         pClaw->field_BC_sprite_scale = field_BC_sprite_scale;
-        pClaw->field_10_anim.field_C_layer = field_10_anim.field_C_layer + 1;
+        pClaw->field_10_anim.field_C_layer = static_cast<Layer>(field_10_anim.field_C_layer + 1);
```

Comment:
can't we know the value from whatever Animation_Init_417FD0 set? (very odd btw because it seems to ignore the scale and I'm sure these things appear in the background)